src/util-debug.h

```diff
@@ -530,6 +530,8 @@ SCLogInitData *SCLogAllocLogInitData(void);
 
 void SCLogAppendOPIfaceCtx(SCLogOPIfaceCtx *, SCLogInitData *);
 
+void CallbackFatalErrorOnInit(const char *);
```

Comment:
why is this named this way?

Reply:
Couldn't come up w a good name 😅 Do you have suggestions?

Reply:
SCFatalErrorOnInit?